components/cdc/src/metrics.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use lazy_static::*;
+use prometheus::*;
+
+lazy_static! {
+    pub static ref CDC_RESOLVED_TS_GAP_HISTOGRAM: Histogram = register_histogram!(
+        "tikv_cdc_resolved_ts_gap",
+        "Bucketed histogram of the gap between cdc resolved ts and current tso",
+        exponential_buckets(0.0005, 2.0, 20).unwrap()
```

Comment:
```suggestion
        exponential_buckets(0.0005, 2.0, 24).unwrap()
```

The largest bucket of e20 is 8.7min, I'd like to extend it to e24, 139min.